proto/kvrpcpb.proto

```diff
@@ -33,7 +33,7 @@ message KeyError {
 message Context {
     optional uint64             region_id    = 1;
     optional metapb.RegionEpoch region_epoch = 2;
-    optional metapb.Peer        peer         = 3;
+    optional uint64             store_id     = 3;
```

Comment:
`peer` was used in raftkv to identify which peer to send when there are multiple stores. But since raftkv only has one store now, so we don't need it anymore.